app/javascript/components/time_table/timesheet/history/work_hours.js

```diff
@@ -128,6 +132,10 @@ class WorkHours extends React.Component {
     });
   }
 
+  onTagChange(value) {
+    this.setState({ tag: value });
```

Comment:
consider renaming `value` to `tag` so so you can replace it with `this.setState({ tag })`